src/layouts/shortcodes/cloud-prepare/openstack.md

```diff
@@ -0,0 +1,37 @@
+Prepare OpenShift-on-AWS **cluster-a** for Submariner:
+
+<div class="notices note">
+<p>The default EC2 instance type for the Submariner gateway node is <code>c5d.large</code>,
+optimized for better CPU which is found to be a bottleneck for IPsec and Wireguard drivers.
+Alternatively, you can choose to deploy using a different instance type.</p>
+</div>
+
+```bash
+export KUBECONFIG=cluster-a/auth/kubeconfig
+subctl cloud prepare aws --ocp-metadata path/to/cluster-a/metadata.json {{ with $.Get "nattPort" }}--natt-port {{.}}{{ end }}
```

Comment:
What is this natt-port argument?

Reply:
I mean this one `--natt-port uint16            IPSec NAT traversal port (default 4500)` Since the default port may not work with Openstack

Reply:
But the final rendered page does not seem to include this option.

Reply:
Fixed , now using natport directly.